.github/workflows/release-artifacts.yaml

```diff
@@ -25,12 +25,15 @@ jobs:
       uses: olegtarasov/get-tag@v2
       id: tagName
 
-    - name: Build on PR
-      id: docker_build
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.CR_PAT }}
```

Comment:
we prob don't need to share the PAT anywhere, and will invalidate/regenerate if we need?

Reply:
Yes, if we need a new one any owner can do it.

Reply:
And now we don't need that until we migrate to the new registry.